packages/block-editor/src/components/block-preview/README.md

```diff
@@ -29,3 +29,17 @@ A block instance (object) or a blocks array you would like to render a preview.
 Width of the preview container in pixels. Controls at what size the blocks will be rendered inside the preview.
 
 `viewportWidth` can be used to simulate how blocks look on different device sizes or to make sure make sure multiple previews will be rendered with the same scale, regardless of their content.
+
+### `onReady`
+* **Type** `Function`
+* **Default:** `noop`
+
+Use this callback as an opportunity to know when the preview is ready. The callback will pass the scale factor, offsets position (x, y) and the DOM element reference. Eg:
```

Comment:
Should we say:
will pass the scale factor, offsets position (x, y) and the DOM element reference, _if available_

The mobile application will also probably need to use this component and there we may not have a dom element for example.